Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Guns/snipers.yml

```diff
@@ -8,5 +8,89 @@
     whitelist:
       tags:
       - CartridgeLightRifle
-    capacity: 10
+    capacity: 5
     proto: CartridgeLightRifleRubber
+
+- type: entity
+  id: WeaponSniperSVT
+  name: SVT-40
+  parent: BaseWeaponRifle
+  description: Magazine fed rifle chamber in .30 Cal. "Retreating is considered Treason" is carved in the wooden stock of the gun
```

Comment:
.30 Cal is not a bullet type, this should match the name of the round this is supposed to fit exactly as is shown in-game. (Check mosin for an example.) I think this takes `.30 rifle` ammo, but I can't tell from here.

Treason should not be capitalised unless it's a stylistic choice. You're missing a period at the end of the description.